stellar-multitenant/internal/httphandler/tenants_handler_test.go

```diff
@@ -51,11 +54,54 @@ func runBadRequestPatchTest(t *testing.T, r *chi.Mux, url, fieldName, errorMsg s
 	assert.JSONEq(t, expectedRespBody, string(respBody))
 }
 
-func runSuccessfulRequestPatchTest(t *testing.T, r *chi.Mux, ctx context.Context, dbConnectionPool db.DBConnectionPool, handler TenantsHandler, reqBody, expectedRespBody string) {
+func runRequestStatusUpdatePatchTest(t *testing.T, r *chi.Mux, ctx context.Context, dbConnectionPool db.DBConnectionPool, handler TenantsHandler, getEntries func() []logrus.Entry, originalStatus, reqStatus tenant.TenantStatus, errorMsg string) {
+	tenant.DeleteAllTenantsFixture(t, ctx, dbConnectionPool)
+	tnt := tenant.CreateTenantFixture(t, ctx, dbConnectionPool, "aid-org", "GCTNUNQVX7BNIP5AUWW2R4YC7G6R3JGUDNMGT7H62BGBUY4A4V6ROAAH")
+
+	_, err := handler.Manager.UpdateTenantConfig(ctx, &tenant.TenantUpdate{ID: tnt.ID, Status: &originalStatus})
+	require.NoError(t, err)
+
+	url := fmt.Sprintf("/tenants/%s", tnt.ID)
+
+	rr := httptest.NewRecorder()
+	req, err := http.NewRequest(http.MethodPatch, url, strings.NewReader(fmt.Sprintf(`{"status": "%s", "id": "%s"}`, reqStatus, tnt.ID)))
+	require.NoError(t, err)
+	r.ServeHTTP(rr, req)
+
+	resp := rr.Result()
+	defer resp.Body.Close()
```

Comment:
🤔 shouldn't you be closing it after the `readAll` call?

Reply:
I think this is the preferred standard since it comes immediately after the resp is opened? for [example](https://github.com/stellar/stellar-disbursement-platform-backend/blob/eb3903e19e364d759c52da5c61cce07d2070bf93/internal/serve/validators/recaptcha.go#L52-L58)